src/Conversion/KrnlToLLVM/KrnlCall.cpp

```diff
@@ -83,33 +86,36 @@ class KrnlCallOpLowering : public ConversionPattern {
   }
 
 private:
-  static void handleOneParameter(PatternRewriter &rewriter, Operation *op,
+  void handleOneParameter(PatternRewriter &rewriter, Operation *op,
       Value parameter, Value original,
       llvm::SmallVector<Type, 4> &parameterTypeList,
       llvm::SmallVector<Value, 4> &parameterList,
-      llvm::SmallVector<Value, 4> &omTensors) {
+      llvm::SmallVector<Value, 4> &omTensors) const {
     MLIRContext *context = op->getContext();
     Location loc = op->getLoc();
     ModuleOp module = op->getParentOfType<ModuleOp>();
-    const auto &apiRegistry = RuntimeAPIRegistry(module, rewriter);
     MultiDialectBuilder<LLVMBuilder> create(rewriter, loc);
+    LLVMTypeConverter *llvmTypeConverter =
+        static_cast<LLVMTypeConverter *>(getTypeConverter());
```

Comment:
Consider using auto, since you already have the type in the cast
```suggestion
    auto *llvmTypeConverter =
        static_cast<LLVMTypeConverter *>(getTypeConverter());
```